modules/Bio/EnsEMBL/Compara/PipeConfig/Example/EGProteinTrees_conf.pm

```diff
@@ -280,6 +281,7 @@ sub resource_classes {
          '64Gb_job'     => {'LSF' => '-q production-rh6 -M64000 -R"select[mem>64000] rusage[mem=64000]"' },
          '512Gb_job'     => {'LSF' => '-q production-rh6 -M512000 -R"select[mem>512000] rusage[mem=512000]"' },
 
+         '8Gb_8c_job' => {'LSF' => '-q production-rh6 -n 8 -C0 -M8000 -R"select[mem>8000] rusage[mem=8000] span[hosts=1]"' },
```

Comment:
Hi @khowe, sorry I didn't see your reply.
Anyways it was fine and @muffato has approved the pull request.